examples/responseselectorbot/credentials.yml

```diff
@@ -0,0 +1,32 @@
+twilio:
```

Comment:
I don't think these should be in here

Reply:
@tmbo Do you mean the actual credentials shouldn't be in here and we should put some dummy credentials?
I actually assumed these are dummy credentials that don't work because I took them from the moodbot example. We should change it there too if these are actually working credentials.

Reply:
I see what you mean. We are not using channels in the example, so the file `credentials.yml` isn't needed even.